frontend/package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@patternfly/react-core": "^1.35.1",
+    "@patternfly/react-tokens": "1.9.1",
```

Comment:
@MusaTalluzi is this used? How?

Reply:
I remember using it experimentally at some point, but I don't need it any more I'll remove it in the next PR. Thanks for the catch!